server/sonar-web/src/main/js/apps/overview/qualityGate/ApplicationQualityGateProject.js

```diff
@@ -51,11 +54,12 @@ type Props = {
     status: string
   }
 };
+*/
 
 export default class ApplicationQualityGateProject extends React.PureComponent {
-  props: Props;
+  /*:: props: Props; */
 
-  renderCondition = (condition: Condition) => {
+  renderCondition = (condition /*: Condition */) => {
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove parentheses around the parameter of this arrow function. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=javascript%3AS3524)